package.json

```diff
@@ -102,5 +102,8 @@
     "fix": "eslint --fix 'src/**/*.{ts,tsx}'; prettier --config .prettierrc.json --check 'src/**/*.{ts,tsx}'",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "publishStorybook": "storybook-to-ghpages --out=.out"
+  },
+  "dependencies": {
+    "pretty-quick": "^3.1.0"
```

Comment:
This a new dependency and the only one listed under "dependencies". Seems like it is in the wrong place. I think we should move this to devDependencies.

Reply:
so instead of doing

`% yarn add pretty-quick`

do:

`% yarn add --dev pretty-quick`

correct? @TCL735

Reply:
yeah i think so